README.tmpl.md

```diff
@@ -61,14 +61,19 @@ List prerequisites and outline detailed steps in this section for users to setup
 
 <!-- begin template instructions replace -->
 
-Highlight any required [input variables](https://consul-klkdtvr57.vercel.app/docs/nia/configuration#variable_files) or [user-defined metadata](consul.io/docs/nia/configuration#cts_user_defined_meta) and provide an example configuration for Consul Terraform Sync for your module.
+Highlight any required [intermediate input variables](https://consul.io/docs/nia/configuration#variable_files) or [user-defined metadata](https://consul.io/docs/nia/configuration#cts_user_defined_meta). Mention any [Consul Terraform Sync provided input variables](https://consul.io/docs/nia/terraform-modules#optional-input-variables) that are included. Provide an example configuration for Consul Terraform Sync for your module.
```

Comment:
The https://consul.io/docs/nia/terraform-modules#optional-input-variables doesn't work yet since the doc changes haven't been merged. This link can be previewed: https://consul-nzi11ca23-hashicorp.vercel.app/docs/nia/terraform-modules#optional-input-variables